pages/release-process.md

```diff
@@ -144,6 +144,72 @@ When the vote passes on a release candidate, follow the steps below to complete
 
  8.  Send an email to `dev@fluo.apache.org` announcing new release.
 
+### Test a Fluo release
+
+Below are two common testing strategies to verify a Fluo release candidate:
+
+1. Run the integration tests of projects that depend on Fluo (using the release candidate jars)
+
+1. Run example Fluo applications (using the release candidate tarball)
+
+These strategies are described below with step-by-step instructions.
+
+#### Run integration tests
+
+A Fluo release can be tested by running the integration tests of projects that use Fluo.
+
+1. Before running integration tests, you'll need the following information:
+    * Release version (example: `1.2.0`)
+    * Staging repo (example: `https://repository.apache.org/content/repositories/orgapachefluo-1023/`)
+        * Note: The staging repo changes with each release candidate.
+
+1. Add the following configuration to `~/.m2/settings.xml` to instruct Maven to build using the staging repo
+   for the release candidate. Also, make sure that the `fluoRC` profile is active while you are testing the release
+   but comment out `<activeProfiles>` section after you are done testing.
+    ```xml
+    <profiles>
+      <profile>
+        <id>fluoRC</id>
+        <repositories>
+          <repository>
+            <id>fluorc</id>
+            <name>fluorc</name>
+            <url>https://repository.apache.org/content/repositories/orgapachefluo-1023/</url>
+          </repository>
+        </repositories>
+        <pluginRepositories>
+          <pluginRepository>
+            <id>fluorcp</id>
+            <name>fluorcp</name>
+            <url>https://repository.apache.org/content/repositories/orgapachefluo-1023/</url>
+          </pluginRepository>
+        </pluginRepositories>
+      </profile>
+    </profiles>
+    <activeProfiles>
+      <activeProfile>fluoRC</activeProfile>
+    </activeProfiles>
+    ```
+1. Clone a project that uses Fluo like the [Fluo Recipes] project below:
+    ```shell
+    $ git clone https://github.com/apache/fluo-recipes.git
+    ```
+    In addition to [Fluo Recipes], there are several projects containing example Fluo applications
+    that have integration tests:
+    * [Phrasecount] - `https://github.com/astralway/phrasecount.git`
+    * [Stresso] - `https://github.com/astralway/stresso.git`
+    * [Webindex] - `https://github.com/astralway/webindex.git`
+1. Run the integration tests using `mvn verify`.  Make sure to set the `fluo.version` to the release
```

Comment:
Shorten to: `Build with: $ mvn -s /tmp/fluo-its-settings.xml clean verify -Dfluo.version=$RCVERSION`
Get rid of preceding paragraph which is redundant with the command.